huaweicloud/resource_huaweicloud_rds_instance_v3_test.go

```diff
@@ -56,8 +56,45 @@ func TestAccRdsInstanceV3_basic(t *testing.T) {
 	})
 }
 
+func TestAccRdsInstanceV3_withEpsId(t *testing.T) {
+	name := acctest.RandString(10)
+	resourceName := "huaweicloud_rds_instance_v3.instance"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckEpsID(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRdsInstanceV3Destroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRdsInstanceV3_epsId(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRdsInstanceV3Exists(),
+					resource.TestCheckResourceAttr(resourceName, "enterprise_project_id", OS_ENTERPRISE_PROJECT_ID),
+				),
+			},
+		},
+	})
+}
+
 func testAccRdsInstanceV3_basic(val string) string {
 	return fmt.Sprintf(`
+resource "huaweicloud_vpc_v1" "test" {
```

Comment:
suggest to use the resource without version

Reply:
fixed